pkgs/by-name/rp/rpcs3/package.nix

```diff
@@ -148,6 +149,14 @@ stdenv.mkDerivation (finalAttrs: {
     qtwayland
   ];
 
+  patches = [
+    (fetchpatch {
+      name = "fix-build-qt-6.10.patch";
+      url = "https://github.com/RPCS3/rpcs3/commit/038ee090b731bf63917371a3586c2f7d7cf4e585.patch";
```

Comment:
You probably want to add `?full_index=1` to the end of this URL, which will avoid the problem where the upstream patch changes because the git commit hash needs to be longer to be unambiguous. This will also require updating the patch hash.